UnityProject/Assets/Tests/EncryptionTest.cs

```diff
@@ -0,0 +1,29 @@
+using System.Text;
+using NUnit.Framework;
+using Util;
+
+namespace Tests
+{
+	public class EncryptionTest
+	{
+		[Test]
+		public void EncryptionTestFunction()
```

Comment:
call this something that describes what the method is testing. A common pattern I personally like to use is "WhenXYShouldZ"

Reply:
I hope the new function name is good enough

Reply:
uhhhm, you have to treat each method as a test case (which what they are). so something like ``DecryptingWithCorrectKeyResultsInOriginalMessage`` and yes, I know it is long but that's fine for tests. Then you can also do the opposite ``DecryptingWithWrongKeyResultsInNothingLikeTheOriginalMessage`` or something like that.

Reply:
Done 👍